### ISSUE TITLE
Cannot to update to gtdbtk 2.1.1 with conda

### ISSUE THREAD

**Issue by fplaza** [Ecogenomics/GTDBTk#416]
Hi,

I would like to update gtdbtk from version 2.1.0 to 2.1.1 using bioconda.
The update fails with the following error:

```
$mamba update -c bioconda  gtdbtk=2.1.1
Encountered problems while solving:
  - package gtdbtk-2.1.1-pyhdfd78af_0 requires fastani 1.3, but none of the providers can be installed
```

Yet, fastANI 1.32 is correctly installed and working
```
$conda list fastani
# Name                    Version                   Build  Channel
fastani                   1.32                 he1c1bb9_0    bioconda
$fastANI --version
version 1.32
```

I have looked at the meta.yaml file.
Previously, fastANI 1.32 was required ( - fastani <=1.32). Now, version 1.3 is mandatory (- fastani ==1.3).

Is it done on purpose?

Thanks,
Florian

**Comment by aaronmussig** [comment]
Hi Florian, sorry for any inconvenience! We fixed the version to 1.3 as there's been some version instability (https://github.com/ParBLiSS/FastANI/issues/101. Primarily it's because 1.3 is the version we use in the GTDB species clustering workflow, and ANI values can differ slightly. 

Either uninstalling FastANI then rerunning update, or installing it into a new environment (and optionally copying across the ref package) will solve it.

**Comment by fplaza** [comment]
Thanks Aaron!
I will install gtdbtk in a distinct environment.
